src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -179,23 +180,30 @@ g.test('visibility,VERTEX_shader_stage_storage_texture_access')
   });
 
 g.test('multisampled_validation')
-  .desc('Test that multisampling is only allowed with "2d" view dimensions.')
+  .desc(
+    `
+  Test that multisampling is only allowed if view dimensions is "2d" and the sampleType is not
+  "float".
+  `
+  )
   .paramsSubcasesOnly(u =>
     u //
       .combine('viewDimension', [undefined, ...kTextureViewDimensions])
+      .combine('sampleType', kTextureSampleTypes)
   )
   .fn(async t => {
-    const { viewDimension } = t.params;
+    const { viewDimension, sampleType } = t.params;
 
-    const success = viewDimension === '2d' || viewDimension === undefined;
+    const success =
+      (viewDimension === '2d' || viewDimension === undefined) && sampleType !== 'float';
```

Comment:
In the spec, I understand that the sampleType of GPUBindGroupLayoutEntry should not "float" if multisampled is true. But, a validation error is not generated when running this test with "float" as a sampleType.

Reply:
This is a known Chromium issue, see https://crbug.com/dawn/1540 and https://github.com/gpuweb/gpuweb/issues/3364

Reply:
Ah, I see. Thanks!